test/dummy/db/schema.rb

```diff
@@ -39,10 +39,10 @@
     t.integer  "deletions",    limit: 4
   end
 
-  add_index "commits", ["author_id"], name: "index_commits_on_author_id", using: :btree
-  add_index "commits", ["committer_id"], name: "index_commits_on_committer_id", using: :btree
-  add_index "commits", ["created_at"], name: "index_commits_on_created_at", using: :btree
-  add_index "commits", ["stack_id"], name: "index_commits_on_stack_id", using: :btree
```

Comment:
Arf. No it's because I use sqlite in dev. I don't think it's a big deal honestly.